ScreenControl_Unity/Assets/ScreenControl/Service/WebSocketReceiver.cs

```diff
@@ -14,8 +14,52 @@ namespace Ultraleap.ScreenControl.Service
     public class WebSocketReceiver : MonoBehaviour
     {
         public ConcurrentQueue<string> setConfigQueue = new ConcurrentQueue<string>();
+        public ConcurrentQueue<string> configRequestQueue = new ConcurrentQueue<string>();
 
         void Update()
+        {
+            CheckSetConfigQueue();
+            CheckConfigRequestQueue();
+        }
+
+        void CheckConfigRequestQueue()
+        {
+            string content;
+            if (configRequestQueue.TryPeek(out content))
+            {
+                // Parse newly received messages
+                configRequestQueue.TryDequeue(out content);
+                HandleConfigRequest(content);
+            }
+        }
+
+        void HandleConfigRequest(string _content) 
```

Comment:
Can we rework this name to distinguish this (reqesting config _state_) from requesting a Config _change_? It threw me for a bit until I realised which this was.